app/internal/event.py

```diff
@@ -65,22 +73,71 @@ def check_diffs(checked_event: Event,
 
 
 def find_pattern(session, event):
-    all_events_with_same_name = session.query(Event).filter(
-        Event.owner_id == event.owner_id, Event.title == event.title).all()
+    all_events_with_same_name = (
+        session.query(Event)
+        .filter(Event.owner_id == event.owner_id, Event.title == event.title)
+        .all()
+    )
 
     return check_diffs(event, all_events_with_same_name)
 
 
-def get_messages(session: Session,
-                 event: Event,
-                 uninvited_contacts: Set[str]) -> List[str]:
+def get_messages(
+    session: Session,
+    event: Event,
+    uninvited_contacts: Set[str],
+) -> List[str]:
     messages = []
     if uninvited_contacts:
-        messages.append(f'Forgot to invite '
-                        f'{", ".join(uninvited_contacts)} maybe?')
+        messages.append(
+            f"Forgot to invite " f'{", ".join(uninvited_contacts)} maybe?',
+        )
 
     pattern = find_pattern(session, event)
     for weeks_diff in pattern:
-        messages.append(f'Same event happened {weeks_diff} weeks before too. '
-                        f'Want to create another one {weeks_diff} after too?')
+        messages.append(
+            f"Same event happened {weeks_diff} weeks before too. "
+            f"Want to create another one {weeks_diff} after too?",
+        )
     return messages
+
+
+def add_countries_to_db(session: Session) -> None:
+    """
+    Adding all new countries to the "Country" table in the database.
+    Information is based on the "countries" list.
+    (The list is located in app/resources/countries.py)
+    Names are described either as:
+    "Country Name, City Name" or
+    "Country Name" solely.
+    Timezones are described as "Continent/ City Name"
+    for example:
+        name: Israel, Jerusalem
+        timezone: Asia/Jerusalem
+    """
+    for country in countries:
```

Comment:
Where you get `countries` object?

Reply:
from app.resources.countries import countries:)

Reply:
Thanks :)
If it's a resource, maybe save it as a json file?

Reply:
Mmm, do you think it's essential? I tried but had a hard time converting it